python/py-plotly/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-plotly
+version             5.3.1
+revision            0
+
+categories          python graphics
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         {nist.gov:joe.fowler @joefowler} openmaintainer
+
+description         Interactive graphing library for Python.
+long_description \
+    Plotly's Python graphing library makes interactive, publication-quality graphs. Examples of how to \
+    make line plots, scatter plots, area charts, bar charts, error bars, box plots, histograms, \
+    heatmaps, subplots, multiple-axes, polar charts, and bubble charts.
+
+homepage            https://plotly.com/python/
+
+checksums           rmd160  679e2ce0095eb3b9e250ed2d6b100120a4613bc1 \
+                    sha256  6598393e898a9c5ae78397f76f07002ec41fd92e5f746d3b9806248d53885643 \
+                    size    7296601
+
+python.versions     36 37 38 39
```

Comment:
It might be better not to add Python 3.6 here as there is a dependency missing and it's going to be EOL at the end of the year.
```
Warning: port lint --nitpick py36-plotly:
Error: Unknown dependency: py36-tenacity
```

Reply:
Ugh. Good point.